db-engine/src/main/java/com/splicemachine/db/impl/sql/catalog/SYSSCHEMASRowFactory.java

```diff
@@ -265,46 +265,52 @@ public TupleDescriptor buildDescriptor(
                 new ColumnDescriptor("AUTHORIZATIONID", 3, 3,
                         DataTypeDescriptor.getBuiltInDataTypeDescriptor(Types.VARCHAR,  false,  128),
                         null, null, view, viewId, 0, 0, 0),
-                new ColumnDescriptor("DATABASEID", 4, 4,
-                        DataTypeDescriptor.getBuiltInDataTypeDescriptor(Types.CHAR, false, 36),
-                        null, null, view, viewId, 0, 0, 0)
+                //new ColumnDescriptor("DATABASEID", 4, 4,
+                //        DataTypeDescriptor.getBuiltInDataTypeDescriptor(Types.CHAR, false, 36),
+                //        null, null, view, viewId, 0, 0, 0)
```

Comment:
Why is this new column not returned?

Reply:
It's not decided yet, but I think that the schema views should not see the database they're in, since we will not be able to see other databases' schemas with `show schemas`

Reply:
OK, that sounds reasonable.